internal/netop/netop.go

```diff
@@ -16,14 +16,32 @@
  *  limitations under the License.
  */
 
+/*
+ *  Copyright (c) 2022 Avesha, Inc. All rights reserved.
```

Comment:
licence is already there at top. this is duplicate

Reply:
removed it